package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "crontab-plugin",
+  "version": "0.0.1",
+  "private": true,
+  "repository": "git@github.com:openshift/crontab-plugin.git",
+  "license": "Apache-2.0",
+  "scripts": {
+    "check-types": "tsc",
+    "clean": "rm -rf dist",
+    "build": "yarn clean && NODE_ENV=production yarn ts-node node_modules/.bin/webpack",
+    "build-dev": "yarn clean && yarn ts-node node_modules/.bin/webpack",
+    "start": "yarn ts-node node_modules/.bin/webpack serve",
+    "start-console": "./start-console.sh",
+    "i18n-to-po": "node ./i18n-scripts/i18n-to-po.js",
+    "po-to-i18n": "node ./i18n-scripts/po-to-i18n.js",
+    "i18n": "./i18n-scripts/build-i18n.sh && node ./i18n-scripts/set-english-defaults.js",
+    "ts-node": "ts-node -O '{\"module\":\"commonjs\"}'",
+    "lint": "eslint ./src --fix && stylelint \"src/**/*.css\" --allow-empty-input --fix"
+  },
+  "devDependencies": {
+    "@openshift-console/dynamic-plugin-sdk": "0.0.12",
+    "@openshift-console/dynamic-plugin-sdk-internal": "^0.0.11",
+    "@openshift-console/dynamic-plugin-sdk-webpack": "0.0.6",
+    "@patternfly/react-core": "4.235.7",
+    "@patternfly/react-table": "^4.112.6",
+    "@types/node": "^16.11.6",
+    "@types/react": "^17.0.34",
+    "@types/react-helmet": "^6.1.4",
+    "@types/react-router-dom": "^5.3.2",
+    "@types/react-tagsinput": "^3.19.0",
+    "@types/webpack-dev-server": "^4.5.0",
+    "@typescript-eslint/eslint-plugin": "^5.3.0",
+    "@typescript-eslint/parser": "^5.3.0",
+    "comment-json": "^4.1.1",
+    "css-loader": "^6.5.1",
+    "eslint": "^8.1.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.26.1",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-simple-import-sort": "^10.0.0",
+    "pluralize": "^8.0.0",
+    "prettier": "^2.4.1",
+    "react": "17.0.1",
+    "react-dom": "^17.0.1",
+    "react-helmet": "^6.1.0",
+    "react-i18next": "^11.8.11",
+    "react-router": "5.2.0",
+    "react-router-dom": "5.2.0",
+    "resolve-url-loader": "^5.0.0",
+    "sass": "^1.58.0",
+    "sass-loader": "^13.2.0",
+    "style-loader": "^3.3.1",
+    "ts-loader": "^9.2.6",
+    "ts-node": "^10.4.0",
+    "tsconfig-paths-webpack-plugin": "^4.0.0",
+    "typescript": "^4.4.4",
+    "webpack": "^5.68.0",
+    "webpack-cli": "^4.9.1",
+    "webpack-dev-server": "^4.5.0"
+  },
+  "consolePlugin": {
+    "name": "crontab-plugin",
+    "version": "0.0.1",
+    "displayName": "OpenShift CronTab Plugin",
+    "description": "CronTab plugin for OpenShift Console",
```

Comment:
Let's improve the description here to make it clear that it's a sample plugin for a custom resource and describe what capabilities it adds.